.github/workflows/dev-check.yml

```
@@ -39,4 +39,4 @@ jobs:
           sudo apt-get install libsdl2-dev libsdl2-image-dev libsdl2-mixer-dev libsdl2-ttf-dev libfreetype6-dev libportmidi-dev python3-dev
 
       - name: Check dev.py all
-        run: python3 dev.py all
+        run: python3 dev.py all --ci --commit_hash ${{ github.sha }} --branch "${{ github.head_ref || github.ref_name }}"
```

actionlint (1.7.11)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)